### PR TITLE
BD-1140: Add documentation for purchases/product_list endpoint

### PR DESCRIPTION
**Pull request by bre-fitzgerald**
# Pull Request/Issue Resolution

Closes #**BD-1140**

#### Is this change associated with a Braze feature/product release?
- [ ] Yes (__Insert Feature Release Date Here__)
- [ ] No

### CODE REVIEW

**Comment by bre-fitzgerald** [comment]
Hi @johndinh—here's what I put together for the `purchases/product_list` endpoint. Right now the link to the Postman collection is a filler, I'll update it and Postman with the endpoint once this content is finalized.

Feel free to tag anyone else as needed for review :)

File: _docs/_api/endpoints/export/purchases/get_list_product_id.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+This endpoint returns paginated lists of product IDs.
+
+<!---Update button link--->
+{% apiref postman %}https://documenter.getpostman.com/view/4689407/SVYrsdsG?version=latest#4b8515b8-067f-41fd-b213-8bb2d18b1557 {% endapiref %}
+
+## Request parameters
+
+| Parameter | Required | Data Type | Description |
+|---|---|---|---|
+| `page`  | Required | String | The page of your product list that you would like to view. |
```
Comment (johndinh) [inline _docs/_api/endpoints/export/purchases/get_list_product_id.md:26]: Not sure if page is required.  I was able to submit a request in Sweeney without it and I got a response with products.

**Review by johndinh — CHANGES_REQUESTED** [review]
Reviewed and left one question.

@mkigikm could you take a look at this too before it goes out?  Thank you!

File: _docs/_api/endpoints/export/purchases/get_list_product_id.md (hunk 1 above)
Comment (bre-fitzgerald) [reply _docs/_api/endpoints/export/purchases/get_list_product_id.md]: Changed to optional!

File: _docs/_api/endpoints/export/purchases/get_list_product_id.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+| Parameter | Required | Data Type | Description |
+|---|---|---|---|
+| `page`  | Optional | String | The page of your product list that you would like to view. |
+{: .reset-td-br-1 .reset-td-br-2 .reset-td-br-3  .reset-td-br-4}
+
+## Example request
+
+{% raw %}
+```
+https://rest.iad-03.braze.com/purchases/product_list?page=1
```
Comment (mkigikm) [inline _docs/_api/endpoints/export/purchases/get_list_product_id.md:33]: Do we want to use iad-01 for this example? It looks like that's what the others all do to me.

**Review by mkigikm — APPROVED** [review]

**Review by johndinh — APPROVED** [review]
LGTM!

**Review by KellieHawks — APPROVED** [review]
Looks good to me!

**Pull request merged**